### PR TITLE
add wrdcz's peer id to allowedPeers.mainnet.ts

### PR DESCRIPTION
**Pull request by wrdcz**
## Motivation

Describe why this issue should be fixed and link to any relevant design docs, issues or other relevant items.

## Change Summary

Describe the changes being made in 1-2 concise sentences.

## Merge Checklist

_Choose all relevant options below by adding an `x` now or at any time before submitting for review_

- [ ] PR title adheres to the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standard
- [ ] PR has a [changeset](https://github.com/farcasterxyz/hub-monorepo/blob/main/CONTRIBUTING.md#35-adding-changesets)
- [ ] PR has been tagged with a change label(s) (i.e. documentation, feature, bugfix, or chore)
- [ ] PR does not require changes to the [protocol](https://github.com/farcasterxyz/protocol)
- [ ] All [commits have been signed](https://github.com/farcasterxyz/hub-monorepo/blob/main/CONTRIBUTING.md#22-signing-commits)

## Additional Context

If this is a relatively large or complex change, provide more details here that will help reviewers

---

## PR-Codex overview
This PR focuses on adding a new allowed peer for the mainnet. 

### Detailed summary
- Added a new allowed peer with the address `12D3KooWCDxDVkirzdV6Ze8zgDq2i1X3tQLGgwTGRxwuYeHD6XDE` for the mainnet.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/hubble/src/allowedPeers.mainnet.ts [hunk 1]
```diff
@@ -47,6 +47,7 @@ export const MAINNET_ALLOWED_PEERS = [
   '12D3KooWNLshuXk4x2gacFTMEJjMGGFxPjpHg5KcV88KoqJP7dDR', // @bhgomes
   '12D3KooWS39hrt41rMuD5ZfPFMxUVycxVwz7w15CxZVhxP4AZDGr', // @rozhnovskiyigor
   '12D3KooWHXZkdEYWb5RCajv7YbffasjYNSEP1U68DMR1JeZ4QiXH', // @fedecastelli
+  '12D3KooWCDxDVkirzdV6Ze8zgDq2i1X3tQLGgwTGRxwuYeHD6XDE', // @wrdcz
```
Comment (varunsrin) [inline apps/hubble/src/allowedPeers.mainnet.ts:50]: can you please add your farcaster id here?

Comment (deodad) [reply apps/hubble/src/allowedPeers.mainnet.ts:50]: I didn't find anyone with this username. is this your farcaster username? if not can you update

Comment (wrdcz) [reply apps/hubble/src/allowedPeers.mainnet.ts]: Done, I thought I should wrote my github ID here😂

Comment (wrdcz) [reply apps/hubble/src/allowedPeers.mainnet.ts]: > can you please add your farcaster id here?

Done, I thought I should wrote my github ID here😂

**Comment by varunsrin** [comment]
added in a batch pr here: https://github.com/farcasterxyz/hub-monorepo/pull/1072

**Pull request closed without merging**